migrate.py

```diff
@@ -272,8 +281,26 @@ def migrate_tickets(self):
             # need to keep trac # in title to have unique titles
             title = "%s (trac #%d)" % (attributes['summary'], trac_id)
 
-            body = self.fix_wiki_syntax(attributes['description'])
-            body += "\n\nMigrated from %s\n" % urljoin(self.trac_public_url, "/ticket/%d" % trac_id)
+            r=self.get_github_username(attributes['reporter'])
+            if r ==GithubObject.NotSet:
+                rep=attributes['reporter']
+            else:
+                rep='@'+r.login
+            body ='\nreported by: '+rep
+            
+            newCC=[]
+            for u in attributes['cc'].strip().split(', '):
+                if u:
+                    newU=self.get_github_username(u)
+                    if newU is GithubObject.NotSet:
+                        newCC.append(u)
+                    else:
+                        newCC.append('@'+newU.login)
+            if newCC:
+                body += "\ncc: %s"%' '.join(newCC)
+
+            body += '\n\n'+self.fix_wiki_syntax(attributes['description'])
+            body += "\n\nMigrated from %s\n" % urljoin(self.trac_public_url, "ticket/%d" % trac_id)
```

Comment:
add reporter and cc: , with users mapped to github users, so that those users still get notified of ticket changes.